gluecodium/src/main/resources/templates/ffi/FfiCallbacksHandleHeader.mustache

```diff
@@ -31,7 +31,7 @@ extern "C" {
 
 _GLUECODIUM_FFI_EXPORT int32_t {{libraryName}}_library_callbacks_queue_init(bool is_main_isolate);
 _GLUECODIUM_FFI_EXPORT void {{libraryName}}_library_callbacks_queue_release(int32_t isolate_id);
-_GLUECODIUM_FFI_EXPORT bool {{libraryName}}_library_wait_for_callbacks(int32_t isolate_id);
+_GLUECODIUM_FFI_EXPORT int32_t {{libraryName}}_library_wait_for_callbacks(int32_t isolate_id);
```

Comment:
Should return uint8_t. This is what Dart code expects (you did not change that part) and it should be enough for a 3-value enum.